packages/client/lib/rpc/modules/engine.ts

```diff
@@ -396,9 +395,9 @@ export class Engine {
         const validTerminalBlock = await validateTerminalBlock(parent, this.chain)
         if (!validTerminalBlock) {
           const response = {
-            status: Status.INVALID_TERMINAL_BLOCK,
+            status: Status.INVALID,
             validationError: null,
-            latestValidHash: null,
+            latestValidHash: ZERO_VALID_HASH,
```

Comment:
thanks for opening! looks great. just thinking, could you instead use `bufferToHex(zeroes(32))`? don't think we need an extra constant really.